src/Security/Group.php

```diff
@@ -115,6 +115,11 @@ public function getAllChildren()
      */
     public function getCMSFields()
     {
+        $list = Group::get()->exclude('ID', $this->ID);
+        $groups = new ArrayList();
+        foreach ($list as $grp) {
```

Comment:
If you use `$group` here it might be a little faster to read and understand what it is